### PR TITLE
fix collection stats

### LINKED ISSUE #3678: Seems collections statistics are gone on RMRK?
It seems we accidentally disabled collection stats on RMRK where they've been before.
https://beta.kodadot.xyz/rmrk/collection/0a8ce195286c168f19-DONKEY

image

### PR DESCRIPTION
**Pull request by KngZhi**
- fix: uncomment
- fix: show highest sale and total buys
- fix: parseFloat but
- feat: rewrite data structure

**Thank you for your contribution** to the [KodaDot NFT gallery](https://kodadot.xyz).
👇 \_ Let's make a quick check before the contribution.

### PR type

- [x] Bugfix
- [ ] Feature
- [ ] Refactoring

### What's new?

- [x] PR closes #3678
- [ ] Requires deployment <rubick/magick_issue>
- [ ] <brief_description_of_what_I've_added>

### Before submitting Pull Request, please make sure:

- [x] My contribution builds **clean without any errors or warnings**
- [x] I've merged recent default branch -- **main** and I've no conflicts
- [x] I've tried to respect high code quality standards
- [x] I've didn't break any original functionality
- [x] I've posted a screenshot of demonstrated change in this PR

### Optional

- [ ] I've tested it at </rmrk/collection/26902bc2f7c20c546a-1FVG7>
- [ ] I've tested PR on mobile and everything seems works
- [ ] I found edge cases
- [ ] I've written some unit tests 🧪

### Had issue bounty label?

- [x] Fill up your KSM address: [Payout](https://beta.kodadot.xyz/transfer?target=DaWoVNNhBTdqJ5iPN3UcaLeamkPpJeGWdrs8LhvX9TQbadc)

### Community participation

- [ ] [Are you at KodaDot Discord?](https://discord.gg/35hzy2dXXh)

### Screenshot

- [x] My fix has changed **something** on UI; a screenshot is best to understand changes for others.

https://deploy-preview-3695--koda-nuxt.netlify.app/rmrk/collection/e0b9bdcc456a36497a-RMRKBNNRS

image

### CODE REVIEW

**Comment by yangwao** [comment]
@petersopko let's check this and merge it :)

**Comment by petersopko** [comment]
> @petersopko let's check this and merge it :)

tested on couple of collections along with https://app.gc.subsquid.io/beta/rubick/006/graphql, wfm

**Review by petersopko — APPROVED** [review]

**Comment by KngZhi** [comment]
@petersopko could re-merge again.

Just make a push to remove the comment; seems broken the auto-merge.

File: components/rmrk/Gallery/CollectionActivity.vue [hunk 1]
```diff
@@ -117,7 +115,7 @@ export default class CollectionActivity extends mixins(PrefixMixin) {
     const { data } = await this.$apollo
       .query({
         query: collectionStatsById,
-        client: this.urlPrefix,
+        client: 'subsquid',
```
Comment (petersopko) [inline components/rmrk/Gallery/CollectionActivity.vue:118]:
```suggestion
        client: this.client,
```

**Review by petersopko — CHANGES_REQUESTED** [review]
just one thing

**Comment by KngZhi** [comment]
One last update, pls wait for a moment.

**Review by petersopko — APPROVED** [review]
ok

**Comment by petersopko** [comment]
pay 100 usd

**Comment by yangwao** [comment]
😍 Perfect, I’ve sent the payout 
💵 $100 @ 66.19 USD/KSM ~ 1.511 $KSM
🧗 [DaWoVNNhBTdqJ5iPN3UcaLeamkPpJeGWdrs8LhvX9TQbadc](https://kusama.subscan.io/account/DaWoVNNhBTdqJ5iPN3UcaLeamkPpJeGWdrs8LhvX9TQbadc)
🔗 [0x18fe9aa3209781c98877d4ecbf3fdc29904ad5c47c0b2a5392648a7fe003bd8d](https://kusama.subscan.io/extrinsic/0x18fe9aa3209781c98877d4ecbf3fdc29904ad5c47c0b2a5392648a7fe003bd8d)

🪅 Let’s grab another issue and get rewarded! 
🪄 [github.com/kodadot/nft-gallery/issues](https://github.com/kodadot/nft-gallery/issues/)

**Pull request merged**